README.md

```diff
@@ -50,30 +53,37 @@ and instruct `Rpm` to update the meta info:
 ```java
 import com.artipie.rpm.Rpm;
 Rpm rpm = new Rpm(storage);
-rpm.update("test/my.rpm");
+rpm.batchUpdate(new Key.From("rpm-repo"));
```

Comment:
@olenagerasimova it's unclear what `rpm-repo` means in this example, what string should I pass here